project/src/main/java/com/google/sps/data/ProductSearchLibrary.java

```diff
@@ -275,17 +309,27 @@ private static void createProductSetItemAndAddToList(String productSetId, String
       productSetList.add(productSet);
     }
 
-    public static void deleteProduct(String productId)
-    throws IOException {
-        try (ProductSearchClient client = ProductSearchClient.create()) {
+  public static void deleteProduct(String productId)
+      throws IOException {
+      try (ProductSearchClient client = ProductSearchClient.create()) {
+
+         // Get the full path of the product.
+        String formattedName =
+            ProductSearchClient.formatProductName(projectId, computeRegion, productId);
 
-            // Get the full path of the product.
-            String formattedName =
-                ProductSearchClient.formatProductName(projectId, computeRegion, productId);
+        // Delete a product.
+        client.deleteProduct(formattedName);
+        System.out.println("Product deleted.");
+      }
```

Comment:
I think you should either remove the try statement or create a catch so you can tell if deleting the product failed or not